### ISSUE TITLE
Item validation fails if vat percentage is a float number

### ISSUE THREAD

**Issue by niklasmannila** [paytrail/paytrail-php-sdk#94]
**Describe the bug**

Starting from 2024-09-01 it must be possible to set vat percentage as float number since the current vat percent 24 will be changed to 25.5 by the Finnish government. A validation exception is currently thrown when trying to set a float vat percentage for an Item and PaymentRequest using the setVatPercentage -function.

**Steps to Reproduce**

1. Create a new item, for example:
```
declare(strict_types=1);

....

$item = new Paytrail\SDK\Model\Item();
$item
	->setReference('foobar')
	->setStamp('foobar')
	->setDeliveryDate('2024-01-01')
	->setProductCode('productcode')
	->setUnitPrice(1500)
	->setDescription('Product name')
	->setVatPercentage(24.0)
	->setUnits(1);
```

**Expected behaviour**

Item::setVatPercentage should accept float numbers, for example 24.0 and 25.5.

**Actual behaviour**

Causes PHP error:
`Paytrail\SDK\Model\Item::setVatPercentage(): Argument #1 ($vatPercentage) must be of type ?int, float given`

**Comment by Saggre** [comment]
@loueranta-paytrail Is this going to be updated soon? I saw there's a branch for this issue: https://github.com/paytrail/paytrail-php-sdk/commit/5d837d10e3ec8b3dbc95062ee33848b9fefae91e

**Comment by loueranta-paytrail** [comment]
@niklasmannila & @Saggre: Support for the decimals in VAT was released today on 2.7.3
